app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, old_file, new_file = command.split(" ")
```

Comment:
If `command does not have 3 arguments, the code will be broken. Add a check for it before